src/helpers/rgba.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import rgb from './rgb'
+
+type RgbaColor = {
+  red: number,
+  green: number,
+  blue: number,
+  alpha: number,
+}
+
+/**
+ * Returns a string value for the color. The returned result is the smallest possible rgba or hex notation.
+ *
+ * @example
+ * // Styles as object usage
+ * const styles = {
+ *   background: rgba(255, 205, 100, 180),
+ *   background: rgba({ red: 255, green: 205, blue: 100, alpha: 180 }),
+ *   background: rgba(255, 205, 100, 255),
+ * }
+ *
+ * // styled-components usage
+ * const div = styled.div`
+ *   background: ${rgba(255, 205, 100, 180)};
+ *   background: ${rgba({ red: 255, green: 205, blue: 100, alpha: 180 })};
+ *   background: ${rgba(255, 205, 100, 255)};
+ * `
+ *
+ * // CSS in JS Output
+ *
+ * element {
+ *   background: "rgba(255,205,100,180)";
+ *   background: "rgba(255,205,100,180)";
+ *   background: "#ffcd64";
+ * }
+ */
+function rgba(value: RgbaColor | number, green?: number, blue?: number, alpha?: number): string {
+  if (typeof value === 'number' &&
+      typeof green === 'number' &&
+      typeof blue === 'number' &&
+      typeof alpha === 'number') {
```

Comment:
I think this is a perfect place that we can eventual leverage #4. Throw a warning and fall back to `rgb` if `alpha` isn't provided.

For alpha I think the proper fallback is `1`. A fully transparent color in `rgba` tends to only be used in transition animations when going to or from a fully transparent color to a more opaque color. 

As far as defaults for the colors, this is a tricky one. `rgba(244)` could be shorthand for one of `rgba(244, 244, 244)`,  `rgba(244, 0, 0)`, or `rgba(244, 255, 255)`. The last option makes little sense to me as omitting a color probably does not mean you want it at 100%. The first option, while jiving with other shorthands somewhat (like `padding(5px)`, may not be how most think about color. I think if we are going to have a default (which I believe we should), `0` makes the most sense.

I've been thinking a lot about this last question as I work on #4. My thoughts, and I would love everyone's opinion here, is that we should list our internal error handling to errors and warnings specific to the methods we define, offloading the rest to the browser. 

For example `timingFunctions` has a set of timingFunctions, if you provide the name of one that doesn't exist we should throw an error as it is critical to how the mixin works.

However, if I pass a bad backgroundSize to `retinaImage` (like say a typo in the keyword), we should let that bubble up via the browser. Otherwise we will paint ourselves into a corner where we have to maintain error checking for general CSS. Did you provide this mixin bad CSS, did you use an unsupported color keyword. Libraries like Bourbon (or SASS in general) let you pass bad CSS into mixins, we should as well.